tests/resources/serializers/test_marcxml_serializer.py

```diff
@@ -317,15 +255,10 @@ def test_marcxml_serializer_full_record(
     <subfield code="a">11 pages</subfield>
   </datafield>
   <datafield tag="536" ind1=" " ind2=" ">
-    <subfield code="c">755021</subfield>
-    <subfield code="a">Personalised Treatment For Cystic Fibrosis Patients With Ultra-rare CFTR Mutations (and beyond)</subfield>
-  </datafield>
-  <controlfield tag="005">{parse(record["updated"]).strftime("%Y%m%d%H%M%S.0")}</controlfield>
-  <datafield tag="856" ind1="4" ind2=" ">
-    <subfield code="s">8</subfield>
-    <subfield code="z">md5:8bc944dbd052ef51652e70a5104492e3</subfield>
-    <subfield code="u">https://127.0.0.1:5000/records/{recid}/files/test.pdf</subfield>
```

Comment:
I added a file again, I agree is a good test. But only modified the `marcxml` record fixture since this seemed to be mostly relevant to MarcXML. If thought otherwise, I can move it up and update all the other serializer's test